tests/integration/cattletest/core/test_whitelist_default_registries.py

```diff
@@ -88,7 +89,7 @@ def test_container_registry_default(super_client, client, context,
     assert_fields(container, {
         "type": "container",
         "state": "stopped",
-        "imageUuid": "sim:localhost.localdomain:5000/sim:v2",
+        "imageUuid": "sim:localhost.localdomain:5000/v2",
         "firstRunning": None,
```

Comment:
new logic will parse `sim:v2` to `sim:localhost.localdomain:5000/v2`. ( with default registry `localhost.localdomain:5000` specified.